### ISSUE TITLE
Some Localization problems

### ISSUE THREAD

**Issue by sssssLug** [DonBruce64/MinecraftTransportSimulator#1788]
Version:1.16.5-22.14.2
①The mod will only identify the language file named as "en_us", ranther than "based on the Minecraft language locale code for your language", such as "zh_cn" and "zh_tw" which means Chinese, so one translation will disable all other language;
②For some language , the last character per text line in the manual will not display.

**Comment by DonBruce64** [comment]
I'm confused about what you mean about one translation disabling the other languages.  IV will use the language you have in your settings on boot, and if that doesn't exist, it'll default to the internal language.  Are you saying that if you have an en_us file it will ignore zh_cn even if your language is zh_cn?

**Comment by sssssLug** [comment]
Yes.Exactly MTS will ingore zh_cn whether en_us exist.I guess other language file would also be ingored,then everyone has to write his translation into en_us no matter what language he's speaking.

**Comment by DonBruce64** [comment]
Noted.  Will fix, as this is not how the system is supposed to work.

**Comment by sssssLug** [comment]
BTW, I extract my language file from the pack.jar, packing it into a normal-folder-struture(as Minecraft Wiki says) resource pack , putting the pack to the resource pack folder(In fact I put it into a global resource pack folder provided by Openloader,but I think they are equal), and find it doesn't work. Can you make the file compatible with normal format resource pack? I think that might be beneficial for localization dissemination.

**Comment by DonBruce64** [comment]
I'm not doing that.  I JUST took the language files out of the config folder because folks wanted them in the packs so they'd come by default with the mod and not have to be added ad-hoc for all modpacks.  I'm not going back to a distributed system.

**Comment by DonBruce64** [comment]
> disable

Can you send me the file you are trying to use?

**Comment by sssssLug** [comment]
[zh_cn.zip](https://github.com/user-attachments/files/16550029/zh_cn.zip)
Sorry I just found out the message. Is it too late?

**Comment by DonBruce64** [comment]
Fixed this just now.  Should work for you if you put the file in the right spot now.  Turns out it wasn't working right on 1.16.5.

**Comment by DonBruce64** [comment]
Addressed in 22.18.0.
